src/somesy/commands/sync.py

```diff
@@ -3,35 +3,45 @@
 from pathlib import Path
 from typing import Optional
 
+from rich.pretty import pretty_repr
+
 from somesy.cff.core import CFF
+from somesy.codemeta import update_codemeta
 from somesy.core.core import ProjectMetadata, get_project_metadata
 from somesy.pyproject.core import Pyproject
 
 logger = logging.getLogger("somesy")
 
 
 def sync(
+    *,
     input_file: Path,
     pyproject_file: Optional[Path] = None,
     cff_file: Optional[Path] = None,
-    create_cff: bool = True,
+    codemeta_file: Optional[Path] = None,
 ):
     """Sync selected metadata files with given input file.
 
     Args:
         input_file (Path): input file path to read project metadata from.
         pyproject_file (Path, optional): pyproject file to read project metadata from.
         cff_file (Path, optional): CFF file path if wanted to be synced. Defaults to None.
-        create_cff (bool, optional): Create CFF file if does not exist. Defaults to True.
+        codemeta_file (Path, optional): codemeta file path if wanted to be synced. Defaults to None.
     """
     metadata = get_project_metadata(input_file)
-    logger.debug(f"Project metadata: {metadata}")
+    logger.debug(
+        f"Project metadata: {pretty_repr(metadata.dict(exclude_defaults=True))}"
+    )
 
     if pyproject_file is not None:
         _sync_python(metadata, pyproject_file)
 
     if cff_file is not None:
-        _sync_cff(metadata, cff_file, create_cff)
+        _sync_cff(metadata, cff_file)
+
+    # NOTE: codemeta should always be last, it uses (some of) the other targets
+    if codemeta_file is not None:
+        update_codemeta(codemeta_file, cff_file)
```

Comment:
what happens if user sets no_cff_sync flag?

Reply:
Then none is passed in the calling method, exactly like with the other files.